doc/whats_new/v1.6.rst

```diff
@@ -276,6 +276,10 @@ Changelog
 :mod:`sklearn.linear_model`
 ...........................
 
+- |Fix| :class:`linear_model.LinearRegression` sets the `cond` parameter when
+  calling the `scipy.linalg.lstsq` solver.
+  :pr:`30040` by :user:`Antoine Baker <antoinebaker>`.
```

Comment:
This entry should be moved to its own PR-specific file under `doc/whats_new/upcoming_changes` to leverage the new towncrier based changelog system.